server/routes/browser.py

```diff
@@ -27,3 +30,18 @@ def kg_main():
 @bp.route('/<path:dcid>')
 def kg_entity(dcid):
     return render_template('/browser/kg_entity.html', dcid=dcid)
+
+
+@bp.route('/browser2/<path:dcid>')
```

Comment:
consider moving this to a new module (browser2, sv_browser, etc), so that only a path renaming is required to launch? this applies to the api, templates, js, etc, just so fewer file renaming is needed later.

Reply:
done